docs/application/native/guides/location-sensors/maps.md

```diff
@@ -1,5 +1,7 @@
 # Maps and Maps Service
 
+[!NOTE]
+All maps-service APIs have been deprecated since Tizen 8.0 and will be removed after two releases without any alternatives.
```

Comment:
Sorry, I have forgotten to add the sign before the note section. Also there should be a space between the note and the description line.
```suggestion
> [!NOTE]
> All maps-service APIs have been deprecated since Tizen 8.0 and will be removed after two releases without any alternatives.

```